docs/literate/src/files/first_steps/getting_started.jl

```diff
@@ -0,0 +1,242 @@
+#src # Getting started
+
+# Trixi.jl is a numerical simulation framework for conservation laws and
+# is written in the [Julia programming language](https://julialang.org/).
+# This tutorial is intended for beginners in Julia and Trixi.jl.
+# After reading it, you will know how to install Julia and Trixi.jl on your computer,
+# and you will be able to download setup files from our GitHub repository, modify them,
+# and run simulations.
+
+# The contents of this tutorial:
+# - [Julia installation](@ref Julia-installation)
+# - [Trixi.jl installation](@ref Trixi.jl-installation)
+# - [Running a simulation](@ref Running-a-simulation)
+# - [Getting an existing setup file](@ref Getting-an-existing-setup-file)
+# - [Modifying an existing setup](@ref Modifying-an-existing-setup)
+
+
+# ## Julia installation
+
+# Trixi.jl is compatible with the latest stable release of Julia. Additional details regarding Julia
```

Comment:
Is this still true if loop vectorization gets deprecated?

Reply:
In general, yes, so this is ok as-is imho